test/unfluff.coffee

```diff
@@ -57,6 +57,18 @@ suite 'Unfluff', ->
         arrayEq orig.expected.tags.sort(), sortedTags, "#{site}: meta tags didn't match expected value"
         arrayEq sortedTags, dataLazy.tags().sort(), "#{site}: meta tags from partial extraction didn't match expected value"
 
+      else if field == 'links'
+        sortedLinks = data.links.sort()
+        sortedLazyLinks = dataLazy.links().sort()
+        if !orig.expected.links
+          orig.expected.links = sortedLinks
+          fs.writeFileSync("./fixtures/test_#{site}.json", JSON.stringify(orig, null, 4))
+        deepEq orig.expected.links.sort(), sortedLinks, "#{site}: links didn't match expected value"
+        # console.log(sortedLinks)
```

Comment:
I mean the commented out console logs (the part it highlighted was a little confusing).